src/test/java/no/unit/nva/model/instancetypes/BookMonographTest.java

```diff
@@ -1,34 +1,26 @@
 package no.unit.nva.model.instancetypes;
 
 import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.ObjectMapper;
 import no.unit.nva.model.exceptions.InvalidPageRangeException;
 import no.unit.nva.model.exceptions.InvalidPageTypeException;
 import no.unit.nva.model.pages.MonographPages;
 import no.unit.nva.model.pages.Pages;
 import no.unit.nva.model.pages.Range;
-import nva.commons.utils.JsonUtils;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
 import org.junit.jupiter.params.provider.NullSource;
 
-import static java.util.Objects.nonNull;
 import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
-class BookMonographTest {
+class BookMonographTest extends BookInstanceTest {
 
-    public static final String LATIN_NUMERAL_ONE = "i";
-    public static final String LATIN_NUMERAL_TWENTY_EIGHT = "xxviii";
-    public static final String THREE_HUNDRED_AND_NINETY_EIGHT = "398";
+    public static final String BOOK_MONOGRAPH = "BookMonograph";
     public static final String ONE = "1";
     public static final String TWENTY_TWO = "22";
```

Comment:
There is no information of what that value is. Is it a page number? Is it an arbitrary number used in various places? If someone changed the value of the variable, one should change the name as well.